app/src/main/java/org/jellyfin/androidtv/model/itemtypes/LiftedItems.kt

```diff
@@ -36,4 +39,14 @@ class Episode(original: BaseItemDto) : PlayableItem(original) {
 	}
 }
 
-class Movie(original: BaseItemDto) : PlayableItem(original)
+class Person(original: BaseItemPerson) {
+	val id: String = original.id
+	val name: String = original.name
+	val role: String = original.role
+	val type: PersonType = original.personType
+	val primaryImage: ImageCollection.Image? = original.primaryImageTag?.let { ImageCollection.Image(original.id, ImageType.Primary, it) }
```

Comment:
Don't persons support multiple images? In that case we could use the imagecollection

Reply:
I doesn't seem like they do, Persons are weirdly different from all other items. (https://github.com/jellyfin/jellyfin-apiclient-java/blob/36cd46c6682fd9506385f87510e182fa332e5210/library/src/main/java/org/jellyfin/apiclient/model/dto/BaseItemPerson.java)

Reply:
I looked at the network responses in the web version and it looks like the "BaseItemPerson" is not actually the base type "person". It's an object used with minimal data showing the relation between a series/movie and a person.

We should probably rename it

Reply:
Oh whoops. `FullDetailsActivity` uses it to display this row, so I assumed it is the only representation of persons :D Will do the rename later.